src/styles/variables.scss

```
@@ -1,4 +1,31 @@
 // ----------------------------------------------------
+// 缩放系数
+:root,
+page {
```

Stylelint (17.11.1)

[error] Unknown type selector "page" (selector-type-no-unknown)

(selector-type-no-unknown)